package ex02.io;

import java.io.IOException;
import java.io.InputStream;

public class MainEntry {

	public static void main(String[] args) throws Exception {
		
		
		
//		int su1 = 0, su2 = 0;
//		System.out.println("데이터 입력 끝은 ctrl + z 누르세요. ");
//		
//		while((su1 = System.in.read()) != -1) { // read API: -1은 더 이상 읽을 것이 없다는 뜻
//			System.out.println(su1+su2);
//		} 이상한수+이상한수+(su1+su2)
		
//		InputStream is = System.in;
//		
//		try {
//			System.out.print("단일 문자 입력 요망: ");
////			int num = is.read(); // api를 보면 예외가 발생함, '0' ~ '9' : 48 ~57 (아스키 값)
//			
//			int num = is.read() - 48;
//			is.read(); is.read(); // 두개 넣고 문제 해결 완료
//			int num2 = is.read() - 48; 
//					
//					System.out.println(num+num2);
////			System.out.println((char)num);
//		} catch (IOException e) {
//			// TODO Auto-generated catch block
////			System.out.println("user error message");
////			System.out.println(e.getMessage());
//			e.printStackTrace();
//		} // api를 보면 예외가 발생함. 처리해야함
	}

}
